### PR TITLE
create get user api

### PR DESCRIPTION
**Pull request by redbiiddsun**

### CODE REVIEW

**Comment by redbiiddsun** [comment]
@BlackBoxBanner Where r u!!!!!

File: src/controllers/users.controller.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { APIController, PostBody } from "@/types/responseType";
+import prisma from "@/utils/prisma";
+import { User } from "@prisma/client";
+
+type UserExcludePassword = Omit<User, "password">;
+export const usersGetController: APIController<
+    UserExcludePassword | null
```
Comment (BlackBoxBanner) [inline src/controllers/users.controller.ts:7]: remove null pls

**Review by BlackBoxBanner — CHANGES_REQUESTED** [review]

File: src/controllers/users.controller.ts (hunk 1 above)
Comment (redbiiddsun) [reply src/controllers/users.controller.ts:7]: no I wont do it

Comment (redbiiddsun) [reply src/controllers/users.controller.ts:7]: do it by yourself

**Review by BlackBoxBanner — APPROVED** [review]

**Pull request merged**